bin/kernel.o

FAILED: bin/kernel.o
gcc -m32 -Iinclude -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-rtti -fno-exceptions -fno-leading-underscore -Wno-write-strings -fno-stack-protector -std=c++11 -Dsnprintf=snprintf -D_WIN32_WINNT=0x0501 -D_MSC_VER=1600 -o bin/kernel.o -c src/kernel.cpp
In file included from /usr/include/stdint.h:34,
                 from /usr/lib/gcc/x86_64-linux-gnu/9/include/stdint.h:9,
                 from src/include/gdt.h:4,
                 from src/kernel.cpp:2:
/usr/include/bits/stdint-intn.h:24:18: error: conflicting declaration ‘typedef __int8_t int8_t’
   24 | typedef __int8_t int8_t;
      |                  ^~~~~~
In file included from src/kernel.cpp:1:
src/include/types.h:4:38: note: previous declaration as ‘typedef char int8_t’
    4 |     typedef char                     int8_t;
      |                                      ^~~~~~
[2/2] gcc -m32 -Iinclude -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-rtti -fno...nprintf=snprintf -D_WIN32_WINNT=0x0501 -D_MSC_VER=1600 -o bin/gdt.o -c src/gdt.cpp
ninja: build stopped: subcommand failed.

--- a/src/include/gdt.h
+++ b/src/include/gdt.h
@@ -2,6 +2,7 @@
 #define __GDT_H
 
 #include <stdint.h>
+#define _STDINT_H
 
     class GlobalDescriptorTable
     {

